_haikus/doggos.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Jellyfish Cloud
+author: Valerdi
+---
+
+beauty in doggos<br>
+puppers,poofs<br>
+Good bois.<br>
```

Comment:
The haiku needs a few changes to be aligned with haiku style. Haikus have a certain number of syllables per line. The first line should have 5 syllables, the second line should have 7 syllables, and the third line should have 5 syllables. 

Right now, your first line is great, it has 5 syllables. But the second and third lines need a few more syllables. 

You can check to make sure you have the right number of syllables by using a tool such as http://www.haikusyllablecounter.com/.

Once you've updated the haiku to follow this format, we can merge it!